### DISCUSSION TITLE
使用 div + grid 布局替代 table 的可能性

### DISCUSSION THREAD

**Discussion by guaijie** [ant-design/ant-design#46652] [Ideas]
可否应用 div + grid 布局替代 table 来提高渲染效率和解决一些样式问题（例如滚动条的显示与隐藏：https://github.com/ant-design/ant-design/issues/29082）

**Comment by zombieJ** [comment]
Table 的 `virtual` 就是通过 `div` 实现的。理论上通过 `components` 把所有的东西都换掉都行。

**Reply by guaijie** [reply]
通过 components 将所有的东西都换掉会增加用户使用的复杂度，事实上这是组件本身带来的问题。所以是否可以不用注重语义化。直接使用 div 代替 table，提高组件本身的渲染效率和附带解决样式问题
